cmd/hil-vpn-privop/create.go

```diff
@@ -3,14 +3,15 @@ package main
 import (
 	"fmt"
 	"os"
+	"os/exec"
 )
 
 // The actual functionality of the 'create' subcommand. The caller is
 // responsible for validating the arguments. If an error occurs,
 // the program will exit with an error. Otherwise, the generated
 // static key is returned.
 func createCmd(vpnName string, vlanNo, portNo uint16) string {
-	cmd := openVpn("--genkey", "--secret", "/dev/fd/1")
+	cmd := exec.Command("openvpn", "--genkey", "--secret", "/dev/fd/1")
```

Comment:
Not totally related to the PR, but why are we reading from `/dev/fd/1`? Is this using some other opened file descriptor?

Reply:
`/dev/fd/` is a magic directory provided by the kernel that allows a process to access already-open file descriptors by name. openvpn doesn't seem to have a way to specify that it should write the key to stdout, so we use `/dev/fd` to make it happen.

Reply:
Ah right that makes sense.  So rather than saving to a file we're piping the output using that fd

Reply:
That's cool, I decided to see what's on my system.
```
naved::~$ls -l /dev/fd/1
lrwx------ 1 naved naved 64 Nov  7 09:58 /dev/fd/1 -> /dev/pts/6
naved::~$ls -l /dev/stdout 
lrwxrwxrwx 1 root root 15 Oct 19 11:31 /dev/stdout -> /proc/self/fd/1
naved::~$ls -l /proc/self/fd/1
lrwx------ 1 naved naved 64 Nov  7 10:00 /proc/self/fd/1 -> /dev/pts/6
```
pts/6 was my pseudo terminal
@zenhack why don't we just use `/dev/stdout` since it would be more readable rather than using `/dev/fd/1`? ( I am not suggesting we change it here, just curious).

Reply:
No reason really -- I just didn't think of it.

I'm going to go ahead and hit merge; feel free to submit another pr for this.